treeherder/model/models.py

```diff
@@ -189,7 +189,9 @@ class Bugscache(models.Model):
     crash_signature = models.TextField(blank=True)
     keywords = models.TextField(blank=True)
     os = models.CharField(max_length=64, blank=True)
-    modified = models.DateTimeField()
+    modified = models.DateTimeField(null=True)
```

Comment:
I added that in due to the get_or_create modification in treeherder/model/models.py line 761, in order to get the test to pass. Removing the get_or_create as you've suggested results in test failures, but since you said the bug should always exist I can modify the test accordingly.